l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -227,7 +227,23 @@ def getCedPrest(self, cedPrest):
 
             if cedPrest.IscrizioneREA:
                 REA = cedPrest.IscrizioneREA
-                vals['rea_code'] = REA.NumeroREA
+                rea_partners = partner_model.search([
+                    ('rea_code', '=', REA.NumeroREA),
+                    ('id', '!=', partner_id)
```

Comment:
@robyf70 in base a https://github.com/OCA/l10n-italy/pull/1181 qui allora bisognerebbe cercare anche per `rea_office`?

Reply:
@eLBati Direi proprio di si.

Reply:
@robyf70 ok grazie.
@SilvioGregorini pensi di riuscire a portare avanti questa PR?

Reply:
@eLBati Non sarebbe meglio togliere completamente la ricerca per il codice REA? Considera che questo codice non viene inserito da tutti perchè comunque non è obbligatorio, l'unico codice obbligatorio è e rimane la partita iva, io lascerei solo la partita iva come unico criterio di search del cliente/fornitore

Reply:
> io lascerei solo la partita iva come unico criterio di search del cliente/fornitore

Infatti è già così.
Questa parte serve solo per scrivere nel partner i dati REA presenti nell'XML.

Reply:
Ok!